04/style.css

```diff
@@ -0,0 +1,151 @@
+* {
+	padding: 0;
+	margin: 0;
+	box-sizing: border-box;
+}
+.nav {
+	padding: 10px 0;
+	height: 80px;
+}
+.nav__list {
+	position: relative;
+	display: flex;
+}
+.nav__list--item {
+	padding: 10px 20px;
+	margin: 0 20px;
+	font-size: 1.2rem;
+	border-top: 3px solid transparent;
+	cursor: pointer;
+	text-transform: uppercase;
+	list-style: none;
+	transition: 0.3s;
+}
+.nav__list--item a {
+	color: black;
+	text-decoration: none;
+}
+.nav__list--item:hover {
+	border-top: 3px solid rgb(0, 0, 0);
+}
+.menu__display:hover > .nav__under {
+	display: block;
+}
+.nav__under {
+	display: none;
+	position: relative;
+	position: absolute;
+	top: 100%;
+	left: 170px;
+	width: 300px;
+}
+.nav__under--list {
+	width: 300px;
+	padding: 10px;
+	font-size: 1rem;
+	text-align: left;
+	text-transform: capitalize;
+	list-style: none;
+}
+.nav__under--list a {
+	color: rgb(0, 0, 0);
+	text-decoration: none;
+	transition: color 0.3s;
+}
+
+.nav__under--list a:hover {
+	color: rgb(14, 144, 180);
+}
+.arrow-right {
+	position: absolute;
+	top: 14px;
+	left: 255px;
+	font-size: 1rem;
+	transform: rotate(90deg);
+}
+.nav__hamburger {
+	display: none;
+}
+@media (max-width: 599px) {
+	.nav {
+		width: 150px;
+		height: 100vh;
+        
+	}
+	.nav__list {
+		position: relative;
+		display: block;
+		transform: translateX(-100%);
+        transition: transform .3s;
+	}
+	.nav__list--item {
+		border: none;
+		transition: 0.3s;
+	}
+	.nav__list--item a {
+		color: rgb(0, 0, 0);
+
+		transition: color 0.3s;
+	}
+	.nav__list--item:hover {
+		border: none;
+	}
+	.nav__list--item a:hover {
+		color: rgb(255, 255, 255);
+	}
+
+	.nav__under {
+        width: 250px;
+		top: 6%;
+		left: 140px;
+		padding: 0 10px;
+		background-color: rgb(5, 107, 107);
+	}
+	.arrow-right {
+		top: 53px;
+		left: 100px;
+		transform: rotate(0);
+		padding: 0 20px;
+	}
+	.nav__hamburger {
+		display: block;
+		position: relative;
+		width: 20px;
+		height: 25px;
+		border: none;
+		border-top: 5px solid #000000;
+		background-color: transparent;
+		cursor: pointer;
+
+		margin-bottom: 50px;
+		margin: 10px 10px;
+		transition: transform 0.3s;
+	}
+	.nav__hamburger::before,
+	.nav__hamburger::after {
+		content: "";
+		position: absolute;
+		top: 0px;
+		left: 0%;
```

Comment:
Jeśli mamy 0 to nie musimy pisać jednostki bo 0 to 0 w każdej użytej jednostce ;)